apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDevelop.md

```diff
@@ -1,184 +1,50 @@
-### Prerequisites
+# Getting Started with UI Fabric
 
-All you need to get started with Fabric is [Node.js](https://nodejs.org/en/) and npm.
+The UI Fabric library is really easy to use! To get started, make sure you have the latest LTS node.js installed:
 
-#### Node.js and npm
+- https://nodejs.org
 
-- Install **Node.js** (supported versions are 8.x through 10.x) from the [Node.js website](https://nodejs.org/en/).
-- Install [Git](https://git-scm.com/).
-- For code editing we like [Visual Studio Code](https://code.visualstudio.com/).
+After you have downloaded and installed this, simply run this command in a folder in which you want to have the project created:
```

Comment:
Might be good to have a heading before this (but not sure what it would say)

Reply:
just get to the meat already :)